connectors/resteasy/pom.xml

```diff
@@ -19,6 +19,12 @@
 			<groupId>org.jboss.resteasy</groupId>
 			<artifactId>resteasy-jaxrs</artifactId>
 			<version>${resteasy-version}</version>
+			<exclusions>
+				<exclusion>
+					<artifactId>httpclient</artifactId>
+					<groupId>org.apache.httpcomponents</groupId>
+				</exclusion>
+			</exclusions>
```

Comment:
Ah. I thought you started coding for the resteasy 3.x and I didn't want to get the work lost.
Else this could also live in a separate PR - as you like. 

On the issue here: Everyone okay with the workaround? @vinodborole 

No specific date. Release is currently done when sufficient changes piled up. I guess it would be justified to release soon.